libbeat/metric/system/process/process.go

```diff
@@ -426,6 +424,13 @@ func (procStats *Stats) matchProcess(name string) bool {
 // cannot be compiled.
 func (procStats *Stats) Init() error {
 	procStats.logger = logp.NewLogger("processes")
+
+	var err error
+	procStats.host, err = sysinfo.Host()
+	if err != nil {
+		procStats.logger.Warnf("Getting host details: %v", err)
+	}
```

Comment:
I'm not sure I 100% trust the behavior of go-sysinfo here. If we get an error, we may want to just set `procStats.host` to `nil` ourselves.